PROCESSES.md

```diff
@@ -142,48 +143,56 @@ job.
 
 ### Detailed steps
 
+Much of this work is automated by the `prepare-release.sh` script,
+but allows for intervention at any point. This documentation assumes
+the use of `prepare-release.sh`, but that's just for simplicity.
+Everything can be done manually.
+
+Note that only the `set-version` command for `prepare-release` needs
+the package ID to be specified. All other commands work on the basis
+of "whatever versions have been changed since the latest commit on
+GitHub are the ones we want to release".
+
 **Create the release PR**
 
-1. Edit the API catalog (`apis/apis.json`). Find the package you
-want to release (by ID) and edit the version key to the new release
-version. Usually this will be a bump of prerelease (e.g.
-1.0.0-beta03 to 1.0.0-beta04) or a minor version bump (e.g. 1.3.0 to
-1.4.0). You may update any dependencies at the same time.
+1. Run `./prepare-release.sh set-version <api> <version>`, e.g.
+`prepare-release.sh Google.Cloud.Speech.V1 2.0.0-beta01`. This
```

Comment:
set-version is missing from this example.

Reply:
Doh!